### DISCUSSION TITLE
How to determine the theme color?

### DISCUSSION THREAD

**Discussion by yablochnyi** [filamentphp/filament#9103] [Help]
How can I determine the theme color for a user so that I can adjust the logo accordingly?

**Comment by yablochnyi** [comment]
I found it!
->darkModeBrandLogo

**Reply by leandrocfe** [reply]
![image](https://github.com/filamentphp/filament/assets/3833889/778e6614-a55c-4740-9859-b072d123006d)
https://twitter.com/zepfietje/status/1712766593787183571
